### DISCUSSION TITLE
Is this plug in dead?

### DISCUSSION THREAD

**Discussion by SCOTTS948** [milo526/homebridge-tuya-web#449] [General]
Set it up for the first time last July. It's been great, but this week it doesn't work at all. None of my devices show up in Homekit anymore. I've tried everything from reboots, to deleting and reinstalling the plug in from scratch. Even created a new cloud project on the Tuya site. Nothing works. I'm out of guesses and options as to what could be wrong. Unless the plug in just no longer works which would really suck!

**Comment by milo526** [comment]
No, it's just that all the less difficult stuff and stuff that I personally need works. That makes it harder to maintain which logically slows down the process.
My system is still working without a problem 🤷🏼‍♂️

**Comment by mensa84** [comment]
@milo526 Could you please help me with that problem pleae? I don't know how to get started if I can't fix that:
https://github.com/milo526/homebridge-tuya-web/discussions/448
